app/views/orders/new.html.slim

```diff
@@ -0,0 +1,63 @@
+- if @event.has_expired?
+  = render partial: "error", locals: { error: "This event is no longer available." }
+- else
+  .container-fluid.tickets-buy
+    = render partial: "header", locals: { event: @event }
+    = form_for [@event, @order] do |f|
+      = f.hidden_field :event_id, value: @event.id
+      .row
+        .col-md-11.col-md-offset-1
+          table.table
+            thead
+              th
+                |  Ticket Type
+              th.center
+                |  Unit Price
+              th.center
+                |  Quantity
+            tbody
+            - f.object.order_items.each do |order_item|
+              tr
+                td.ticket-name
+                  = order_item.ticket_type.name
+                td.ticket-price.center
+                  = formatted_price(order_item.ticket_type.price)
```

Comment:
`ticket_type_price` and `ticket_type_name`, or just get rid of delegates altogether, up to you